src/extlibs/tracy.BUILD

```diff
@@ -0,0 +1,16 @@
+# Description
+# Tracy profiler is a real-time, nanosecond resolution profiler. It is useful to profile our code when using debug
+# symbols would slow down the code too much.
+cc_library(
+    name = "tracy",
+    srcs = ["public/TracyClient.cpp"],
+    hdrs = glob(
+        [
+            "public/**/*.h",
+            "public/**/*.hpp",
+            "public/**/*.cpp",
+        ],
+    ),
+    includes = ["public/tracy"],
+    visibility = ["//visibility:public"],
+)
```

Comment:
Should we compile with `--copt=-O3`? Or is it already pretty fast like this?

Reply:
generally I don't think the Tracy overhead is affected by the compile-time optimizations. We should be running with `--copt=-O3` since it's the most realistic optimizations

Reply:
the `-o` propagates it to all the cpp files if I understand correctly

Reply:
actually you're right, for a second I thought that the build file was for compiling the Tracy GUI.